src/count-islands/index.js

```diff
@@ -1,3 +1,114 @@
-function countIslands(grid) {}
+function countIslands(grid) {
+
```

Comment:
format your code properly, remove the unnecessary white spaces

Reply:
i have removed the white spaces .

Reply:
if you use vscode editor, you can use prettier plugin to format your code